core/domain/taskqueue_services_test.py

```diff
@@ -59,6 +59,36 @@ def test_exception_raised_when_email_task_params_is_not_serializable(self):
                 params,
                 0)
 
+    def test_defer_makes_the_correct_request(self):
+        correct_fn_identifier = '/task/deferredtaskshandler'
+        correct_url = feconf.TASK_URL_DEFERRED
+        correct_queue_name = taskqueue_services.QUEUE_NAME_EMAILS
+        correct_args = (1, 2, 3)
+        correct_kwargs = {'a': 'b', 'c': 'd'}
+        correct_payload = {
+            'fn_identifier': correct_fn_identifier,
+            'args': correct_args,
+            'kwargs': correct_kwargs
+        }
```

Comment:
These variable all start with `correct_`, but there are two different kinds here:

1. The arguments we pass to `defer()`
2. The arguments we expect to be passed to `create_http_task`

Could you separate these two types somehow? Maybe rename the type (2) variables to start with `expected_` instead?